tests/gis_tests/test_geoforms.py

```diff
@@ -51,9 +51,10 @@ def test_geom_type(self):
         # By default, all geometry types are allowed.
         fld = forms.GeometryField()
         for wkt in ('POINT(5 23)', 'MULTIPOLYGON(((0 0, 0 1, 1 1, 1 0, 0 0)))', 'LINESTRING(0 0, 1 1)'):
-            # `to_python` uses the SRID of OpenLayersWidget if the converted
-            # value doesn't have an SRID itself.
-            self.assertEqual(GEOSGeometry(wkt, srid=fld.widget.map_srid), fld.clean(wkt))
+            with self.subTest(wkt=wkt):
+                # `to_python` uses the SRID of OpenLayersWidget if the converted
```

Comment:
comments should be wrapped at 79 chars. You can use to_python() rather than the current backticked version